source/scoped-mutex-traits/Cargo.toml

```
@@ -1,4 +1,22 @@
 [package]
 name = "scoped-mutex-traits"
-version = "0.1.0"
+version = "1.0.0"
+authors = [
+    "James Munns <james@onevariable.com>",
+    "Eliza Weisman <eliza@elizas.website>",
+]
 edition = "2021"
+readme = "../../README.md"
+repository = "https://github.com/tosc-rs/scoped-mutex"
+description = "An abstraction over closure-based mutexes"
+license = "MIT OR Apache-2.0"
+categories = [
+    "embedded",
+    "no-std",
+]
+documentation = "https://docs.rs/scoped-mutex/"
```

this crate should probably link to its own docs instead of the other crate's?
```suggestion
documentation = "https://docs.rs/scoped-mutex-traits/"
```